Tests/MacroToolkitTests/MacroToolkitTests.swift

```diff
@@ -678,4 +678,221 @@ final class MacroToolkitTests: XCTestCase {
+    }
+    
+    func testNormalizationFunction() {
+        let type: `Type` = "(testParameter: [Int]) -> Void"
+        
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "(testParameter: Array<Int>) -> ()")
+    }
+    
+    func testNormalizationImplicitylUnwrappedOptional() {
+        let type1: `Type` = "MyClass<[String]>!"
+        let type2: `Type` = "((inout [String: Int]) -> Void)!"
+
+        let normalizedType1 = type1.normalized()
+        let normalizedType2 = type2.normalized()
+
+        XCTAssertEqual(normalizedType1.description, "MyClass<Array<String>>!")
+        XCTAssertEqual(normalizedType2.description, "((inout Dictionary<String, Int>) -> ())!")
+    }
+    
+    func testNormalizationMemeberWithGeneric() {
+        let type: `Type` = "(inout TestClass.TestMemberStruct<[any Hashable]>) -> ()"
+        
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "(inout TestClass.TestMemberStruct<Array<any Hashable>>) -> ()")
+    }
+    
+    func testNormalizationMetatype() {
+        let type: `Type` = "(inout TestClass.Type) -> Void"
+        
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "(inout TestClass.Type) -> ()")
+    }
+    
+    func testNormalizationOptional() {
+        let type: `Type` = "((inout Int?) -> Void)?"
+        
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "Optional<((inout Optional<Int>) -> ())>")
+    }
+    
+    func testNormalizationPackExpansion() {
+        let type1: `Type` = "(repeat each Elements)"
+        let type2: `Type` = "(keyPath: KeyPath<(repeat each Elements), Value>) -> Value"
+
+        let normalizedType1 = type1.normalized()
+        let normalizedType2 = type2.normalized()
+
+        XCTAssertEqual(normalizedType1.description, "(repeat each Elements)")
+        XCTAssertEqual(normalizedType2.description, "(keyPath: KeyPath<(repeat each Elements), Value>) -> Value")
+    }
+    
+    func testNormalizationPackReference() {
+        let type: `Type` = "Tuple<each Elements>"
+
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "Tuple<each Elements>")
+    }
+    
+    func testNormalizationSimple() {
+        let type: `Type` = "MyClass<[String]>"
+        
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "MyClass<Array<String>>")
+    }
+    
+    func testNormalizationSuppressed() {
+        let type: `Type` = "~Copyable"
+                
+        let normalizedType = type.normalized()
+
+        XCTAssertEqual(normalizedType.description, "~Copyable")
+    }
+    
+    func testNormalizationAttributedTuple() {
+        let declSyntax: DeclSyntax = """
+        let interestingType: (inout [Int], String) -> Float = { _,_ in
```

Comment:
Add a space after the comma in `_, _` to match the repo's code style (even thought it's not actually code).

Reply:
Fixed in 84c2bd6.